src/NzbDrone.Core/Indexers/BitMeTv/BitMeTvSettings.cs

```diff
@@ -47,6 +52,9 @@ public BitMeTvSettings()
         [FieldDefinition(4, Type = FieldType.Textbox, Label = "Minimum Seeders", HelpText = "Minimum number of seeders required.", Advanced = true)]
         public int MinimumSeeders { get; set; }
 
+        [FieldDefinition(5, Type = FieldType.Textbox, Label = "Seed Ratio", HelpText = "The ratio a torrent should reach before stopping, empty is download client's default")]
+        public string SeedRatio { get; set; }
```

Comment:
This should be stored as a double instead of a string we convert to a double when needed.